src/accelerate/accelerator.py

```diff
@@ -1243,14 +1243,17 @@ def prepare_model(self, model: torch.nn.Module, device_placement=None):
                 has_hf_device_map = True
                 break
 
-        if getattr(model, "is_loaded_in_8bit", False) and getattr(model, "hf_device_map", False):
+        if (getattr(model, "is_loaded_in_8bit", False) or getattr(model, "is_loaded_in_4bit", False)) and getattr(
```

Comment:
Isn't there an `is_quantized` attribute that groups those two? @younesbelkada

Reply:
yes there is, but since it has been introduced for transformers >= 4.30.0 maybe better to keep that logic for backward compatibility